plugins/cost-insights/package.json

```diff
@@ -32,7 +32,6 @@
   "dependencies": {
     "@backstage/config": "^0.1.2",
     "@backstage/core": "^0.4.3",
-    "@backstage/test-utils": "^0.1.5",
```

Comment:
oh sorry; this PR overlapped with #3943, must've had the two branches separate and forgot i did one and the other before!!  if that merges first, can reduce the changeset for this cc @timbonicus